Source/WebCore/rendering/BorderPainter.h

```diff
@@ -76,5 +76,6 @@ class BorderPainter {
 };
 
 LayoutRect shrinkRectByOneDevicePixel(const GraphicsContext&, const LayoutRect&, float devicePixelRatio);
+RoundedRect calculateAdjustedInnerBorder(const RoundedRect&innerBorder, BoxSide);
```

Comment:
This could be static function in the BorderPainter scope.

Reply:
It is being used re-used by RenderBox